src/Translator.php

```diff
@@ -214,12 +214,12 @@ private function parseParameters(array $parameters): array
         }
 
         $params = array_key_exists(1, $parameters) ? $parameters[1] : [];
-        if (!isset($params['count'])) {
+        if ($parameters[0] && !isset($params['count'])) {
             $params['count'] = $parameters[0];
         }
 
         return [
-            'count' => $parameters[0],
+            'count' => $parameters[0] ?? 1,
```

Comment:
ak neposleme count tak bude 1? Ked uz tak 0 nie? alebo null

Reply:
hmmm ok jednotne by to byt urcite malo, ale preco je tam 1 tak to teda neviem :D Spytame sa @ricco24

Reply:
Jasne ze default je 1. Preklad bez poslaneho parametra chceme prekladat ako count = 1.

Reply:
Dakujem za vysvetlenie :D